README.md

```diff
@@ -162,6 +162,6 @@ If you like this project and to give advise am happy to meet you
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
-
+https://github.com/kirube1992/Hello-Microverse/compare/main...add-license-1?quick_pull=1
```

Comment:
- [ ] As requested by the previous reviewer, please kindly ensure that there is no broken link in your readme file.
- [ ] Currently, because there is no LICENSE path in your repo, the link `MIT` leads to an error page.

![image](https://user-images.githubusercontent.com/78251927/233394114-0104424d-04ab-4a02-bcbf-0a352c4bb1b2.png)

![image](https://user-images.githubusercontent.com/78251927/233393644-c41d7ced-3696-4c52-8abc-aa9afa6a63a3.png)

- [ ] I noticed that you included a link to the license of your `Hello Microverse` project. Kindly check here on [how to add a license to a repository](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 💯 🤗

- [ ] Public repositories on GitHub are often used to share open-source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software.